snapshot/snapshot.go

```diff
@@ -306,8 +306,8 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 
 		if !localOverride {
 			if err := o.prepareRemoteSnapshot(lCtx, key, base.Labels); err != nil {
-				log.G(lCtx).WithField(remoteSnapshotLogKey, prepareFailed).
-					WithError(err).Warn("failed to prepare remote snapshot")
+				log.G(lCtx).WithField(remoteSnapshotLogKey, prepareFailed).WithField("context", err).
+					Warn("failed to prepare remote snapshot")
```

Comment:
Are the `err`s (changed from `WithError(err)` -> `WithField("context", err)`) are all noises (e.g., all `nil`, or `<nil>` or not important/repeated message?)

Reply:
The usual `err` returned from `Mount` is `FUSE mount: no ztoc for layer` which is repeated for all layers that do not have ztoc.

Reply:
I think that makes sense, I'll add it